.github/workflows/e2e-tests.yml

```diff
@@ -107,7 +107,7 @@ jobs:
       - name: Install Playwright deps
         shell: bash
         working-directory: grafana-plugin
-        run: yarn playwright install
+        run: pnpm playwright install
```

Comment:
does this need to be `pnpm exec`?

Reply:
actually I think it's the same as just `pnpm`, let me update